### PR TITLE
Integrate AutoRound v0.3 to 2x

### PR DESCRIPTION
**Pull request by Kaihui-intel**
## Type of Change

feature 

## Description

```python
modify:
        n_samples -> nsamples
        n_blocks -> nblocks
quant config add:
        act_bits: int = 32,
        act_group_size: int = None,
        act_sym: bool = None,
        act_dynamic: bool = True,

other config:
        multimodal:bool = False,
        use_layer_wise: bool = False, ->  low_cpu_mem_usage: bool = False, (AutoRound repo)
```

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: neural_compressor/adaptor/torch_utils/weight_only.py [hunk 1]
```diff
@@ -717,15 +724,19 @@ def autoround_quantize(
         sym (bool): Whether symmetric quantization is to be used (default is False).
         weight_config (dict): Configuration for weight quantization (default is an empty dictionary).
         weight_config={
```
Comment (WeiweiZhang1) [inline neural_compressor/adaptor/torch_utils/weight_only.py:726]: layer_config, please update above autoround layer_config arg too.

File: neural_compressor/adaptor/torch_utils/weight_only.py [hunk 2]
```diff
@@ -737,7 +748,7 @@ def autoround_quantize(
         enable_minmax_tuning (bool): Whether to enable weight min-max tuning (default is True).
         lr (float): The learning rate (default is None, will be set to 1.0/iters).
         minmax_lr (float): The learning rate for min-max tuning (default is None, it will be set to lr automatically).
-        low_gpu_mem_usage (bool): Whether to use low GPU memory (default is True).
+        low_gpu_mem_usage (bool): Whether to use low GPU memory (default is False).
         iters (int): Number of iterations (default is 200).
         seqlen (int): Data length of the sequence for tuning (default is 2048).
         n_samples (int): Number of samples (default is 512).
```
Comment (WeiweiZhang1) [inline neural_compressor/adaptor/torch_utils/weight_only.py:754]: nsamples

**Review by WeiweiZhang1 — APPROVED** [review]

**Comment by thuang6** [comment]
It is v0.2.1 instead of v2.1

**Review by changwangss — APPROVED** [review]

**Comment by chensuyue** [comment]
ITREX fix will be submit soon.

**Pull request merged**